code/game/machinery/computer/camera.dm

```diff
@@ -284,39 +303,36 @@
 
 // Other computer monitors.
 /obj/machinery/computer/security/telescreen
-	name = "\improper Telescreen"
+	name = "telescreen"
 	desc = "Used for watching camera networks."
-	icon = 'icons/obj/stationobjs.dmi'
-	icon_state = "telescreen"
+	icon_state = "telescreen_console"
+	icon_screen = "telescreen"
+	icon_keyboard = null
 	light_range_on = 0
-	network = list("SS13")

```

Comment:
I removed it because as it was coded, if you remove a telescreen and then rebuild it, you would get full access to the SS13 network. That seemed like it would have caused problems.